src/components/SpaceSidebarHeader.vue

```diff
@@ -40,7 +40,12 @@ watchEffect(() => {
 
 <template>
   <div class="text-center border-b bg-skin-header-bg h-[253px]">
-    <Token :space="space" symbolIndex="space" size="80" class="mt-3 mb-2" />
+    <SpaceAvatar
+      :space="space"
+      symbolIndex="space"
+      size="80"
+      class="mt-3 mb-2"
```

Comment:
Vue question - Where do these class go an apply when we pass class down the component?

Reply:
To the first element inside the component 
<img width="659" alt="image" src="https://user-images.githubusercontent.com/51686767/156924851-46d6dcc9-4f2c-4abd-b847-80e05d104361.png">